src/main.zig

```diff
@@ -1,12 +1,11 @@
 const std = @import("std");
-const gpu = @import("gpu.zig");
 const builtin = @import("builtin");
+const gpu = @import("mach-gpu");
+const vulkan = @import("vulkan.zig");
 
-var didInit = false;
+var inited = false;
 var allocator: std.mem.Allocator = undefined;
 
-const Implementation = @import("vulkan/impl.zig");
```

Comment:
This line was here to allow the code to more easily be shared between backends, where you would just reimplement the `Implementation` struct, and then all the other code inside `Interface` gets shared between d3d12, vk, mtl, etc.

Reply:
yes. but we are not doing this yet as far as i can say

Reply:
We aren't at the moment, but id rather do the pattern early on than try to bolt it on after the fact to get the codebase ready for the next backend (once vk is in a usable state and we are ready to start another one)